dragonchain/webserver/lib/dragonnet.py

```diff
@@ -80,13 +87,51 @@ def process_receipt_v1(block_dto: Dict[str, Any]) -> None:
                 _log.exception("matchmaking add_receipt failed!")
             # Update the broadcast system about this receipt
             broadcast_functions.set_receieved_verification_for_block_from_chain_sync(l1_block_id, level_received_from, block_model.dc_id)
+            # Send verifications to any configured urls
+            send_verification_notifications(level_received_from, block_model)
```

Comment:
Not sure if this is the best place to put this. This means that any time a dragon net node sends a receipt to an L1, that http request is now hung up on that L1 making a POST to another unrelated server. I can forsee potential timeout issues here. Blocking a webserver request on another arbitrary POST seems like a potentially bad idea

Reply:
(26 days later) Agreed. 
haha. 
After the release of billing, I can jump back into this. Yes. I agree. This PR has been updated to move this logic into the broadcast processor.